### PR TITLE
Add the initial structure for app and database of Chaos engine

### PR DESCRIPTION
**Pull request by spirosoik**
#### Summary
Chaos engine is a Mattermost app with which you can create Gamedays and teams.
The commands give you the ability to:
- Create a team with Mattermost Users
- Schedule a GameDay with a Team
- List Gamedays and Teams

#### Ticket Link
Ticket: https://mattermost.atlassian.net/browse/MM37269

### CODE REVIEW

File: gameday/service.go [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+		return []LookupTeamDTO{}, errors.Wrap(err, "failed to get teams in repository")
+	}
+	var results []LookupTeamDTO
+	for _, team := range teams {
+		results = append(results, team.toLookupTeamDTO())
+	}
+	return results, nil
+}
+
+// LookupTeams responsible to
```
Comment (angeloskyratzakos) [inline gameday/service.go]: Is that explanation complete? same on above func

**Review by angeloskyratzakos — APPROVED** [review]
LGTM nice work Spiros!

File: gameday/repository.go [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+	sql = fmt.Sprintf(sql, id)
+	var teamMembers []TeamMember
+	if err := r.store.DB.Select(&teamMembers, sql); err != nil {
+		return []TeamMember{}, errors.Wrap(err, "failed to list team members")
+	}
+	return teamMembers, nil
+}
+
+// ListTeams returns the list of gamedays created in the app
```
Comment (stafot) [inline gameday/repository.go]: vvvv minor: typo

**Review by stafot — APPROVED** [review]
lgtm

**Pull request merged**